### PR TITLE
Link to embed pages. Remove license headings

### PR DESCRIPTION
**Pull request by Jonas-C**
Fixes https://github.com/NDLANO/Issues/issues/3481
Fixes https://trello.com/c/idJf0mFH/175-fjerne-muligheten-til-%C3%A5-%C3%A5pne-elementer-med-copyright-lisens-i-egen-fane

### CODE REVIEW

**Comment by gunnarvelle** [comment]
I https://ndla-frontend-pr-1412.ndla.sh/subject:1:19dae192-699d-488f-8218-d81535ce3ae3/topic:2:179373/topic:2:170165/resource:1:168358 vises teksten `unknown` i stedet for lisens fordi forklaringa ikkje har lisens. Det ser litt rart ut.

File: src/components/license/licenseHelpers.ts [hunk 1]
```diff
@@ -25,3 +26,6 @@ export const licenseCopyrightToCopyrightType = <T extends BaseCopyright>(
     creators,
   };
 };
+
+export const isCopyrighted = (license?: string) =>
+  license?.toLowerCase() === COPYRIGHTED;
```
Comment (gunnarvelle) [inline src/components/license/licenseHelpers.ts:31]: Her skulle det kanskje være toUpperCase?

Comment (Jonas-C) [reply src/components/license/licenseHelpers.ts:31]: Nei, dette er en constant fra `@ndla/licenses`.

File: src/components/license/VideoLicenseList.tsx [hunk 2]
```diff
@@ -48,7 +53,18 @@ const VideoLicenseInfo = ({ video }: VideoLicenseInfoProps) => {
   return (
     <MediaListItem>
       <MediaListItemImage>
-        <img alt="presentation" src={video.cover} />
+        {isCopyrighted(video.copyright?.license.license) ? (
+          <img alt="presentation" src={video.cover} />
```
[the remark this answers is not in the document]
Comment (Jonas-C) [reply src/components/license/VideoLicenseList.tsx:57]: Mja. Vi har video-tittel. That's it.

File: src/components/license/licenseHelpers.ts (hunk 1 above)
Comment (gunnarvelle) [reply src/components/license/licenseHelpers.ts:31]: Ah. La ikkje merke til at det mangla fnutter.

**Pull request merged**